config/user_preferences_extra_conf.yml.sample

```diff
@@ -0,0 +1,41 @@
+preferences:
+    # the key you can refer to
+    apollo_url_01:
```

Comment:
These are available by default, when it's not quite apparent why the user can enter apollo and openstack information. Should we maybe comment the first 2 sections ?